client/state/action-types.js

```diff
@@ -258,6 +258,8 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 	'HELP_TICKET_CONFIGURATION_REQUEST_FAILURE';
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
+export const HTTP_DATA_REQUEST = 'HTTP_DATA_FETCH';
+export const HTTP_DATA_TICK = 'HTTP_DATA_TICK';
```

Comment:
What's the purpose this TICK action is serving? Is it to force updates for redux connected components?

Reply:
you got it. without dispatching then none of the listeners will know things updated